doc/custom_conf.py

```diff
@@ -172,3 +173,7 @@
 # can be copied even if they don't contain an EOF line.
 copybutton_prompt_text = '$ '
 copybutton_only_copy_prompt_lines = False
+
+extlinks = {
+    'manualpage': ( 'https://manpages.ubuntu.com/manpages/lunar/en/%s', '' )
```

Comment:
Ah, yeah :)

Reply:
Hmm, the thing is... those pages are not yet up, so the linkcheck would fail ;) (The internal nginx redirect handles it by redirecting back to the Lunar pages :)

Reply:
OK